WalletWasabi.Documentation/WasabiCompatibility.md

```diff
@@ -0,0 +1,34 @@
+# Abstract
+
+This document lists all the officially supported software and devices by Wasabi Wallet. This means that Wasabi makes tests on those systems and put all the efforts to make it work and maintain compatibility. One of our main goals is to not break the user space so we have to set up boundaries that we can responsibly maintain. This does not necessarily mean that systems that are not listed will not work - they might work but we not give support for them. There are a lot of systems out there and more to come we have to focus our priorities.
+
+# Officially Supported Operating systems
+
+- Windows 10
+- macOs 10.13+
+- Ubuntu 16.04+
+- Fedora 30+
+- Debian 9+
+- CentOS 7+
+
+# Officially Supported Hardware wallets
+
+- ColdCard MK1
+- ColdCard MK2
+- ColdCard MK3
+- Ledger Nano S
```

Comment:
should we add trezor here?

Reply:
I will add it to the FAQ

Reply:
We'll add Trezor to the FAQ section. Also Whonix+Tails.